Fastlane/shared_lanes.rb

```diff
@@ -113,7 +113,11 @@ def slack_message(message, options = {})
     message: slack_message,
     success: slack_success,
     default_payloads: default_payloads,
-    payload: slack_payload
+    payload: slack_payload,
+    attachment_properties: {
+      author_name: "Fastlane (iOS)",
+      author_icon: "https://upload.wikimedia.org/wikipedia/commons/thumb/8/84/Apple_Computer_Logo_rainbow.svg/206px-Apple_Computer_Logo_rainbow.svg.png"
```

Comment:
Would it be possible to set the icon in the Webhook configuration? To check it, you might need to open a ticket for IT.

I agree the reasoning is great, it might be better to have it fixed. What do you think?

Reply:
I'm not sure what the advantage of that would be? Wouldn't we take away flexibility from us this way?

Reply:
> I'm not sure what the advantage of that would be? Wouldn't we take away flexibility from us this way?

I think the main advantage would be to not rely on a link on the internet.

Reply:
I see.. Well I guess we'll be safe with Wikimedia for a while. If it ever breaks we'll notice rather fast.